crates/matrix-sdk-crypto/src/identities/device.rs

```diff
@@ -642,6 +642,14 @@ impl DeviceData {
         self.local_trust_state() == LocalTrust::BlackListed
     }
 
+    /// Is the device locally marked as whitelisted.
+    ///
+    /// Whitelisted devices will receive group sessions regardless of their
+    /// verification status.
+    pub fn is_whitelisted(&self) -> bool {
+        self.local_trust_state() == LocalTrust::Ignored
```

Comment:
The LocalTrust::Ignored actually is really Whitelist, as it says that the trust state should be ignored. Should it be renamed to Whitelisted? to be consistent with LocalTrust::BlackListed?

Reply:
yeahh, this feels pretty confusing to me, and could do with clean up. 

That said: "whitelisted"/"blacklisted" are somewhat problematic terms, and we might take the opportunity to rename both.